tuareg.el

```diff
@@ -257,6 +257,12 @@ See `ff-other-file-alist'."
   :group 'tuareg
   :type '(repeat (list regexp (choice (repeat string) function))))
 
+(defcustom tuareg-comment-show-paren t
+  "If t, highlight the dilimiters of comments when
```

Comment:
Spelling: "delimiters".
And Emacs standard is to try to cram an entire descriptive sentence on the first line (because it's the one displayed by eldoc etc); might try something like
```
"Highlight comment delimiters in `show-paren-mode' if non-nil."
```